pkg/controller/v1alpha2/applicationconfiguration/render_test.go

```diff
@@ -214,6 +215,7 @@ func TestRenderComponents(t *testing.T) {
 									oam.LabelAppName:              acName,
 									oam.LabelAppComponentRevision: "",
 									oam.LabelOAMResourceType:      oam.ResourceTypeTrait,
+									oam.LabelAppGeneration:        "0",
```

Comment:
I don't see any code will add this label, why the test case will have this label if we don't manuelly add here?